CMakeLists.txt

```diff
@@ -81,11 +81,8 @@ tristate_feature_auto(ENABLE_RIPEMD160 "Enable RIPEMD-160 hash support.")
 
 option(ENABLE_CRYPTO_REFRESH "Enable crypto-refresh support (v6)")
 option(ENABLE_PQC "Enable PQC support - requires ENABLE_CRYPTO_REFRESH")
```

Comment:
Please add check whether crypto-refresh support is enabled only together with PQC, as users could attempt to use it with the unknown result(s).